test/meson.build

```diff
@@ -0,0 +1,22 @@
+include_directories = ['../include']
+
+buffer_test_exe = executable('qa_buffer', 'qa_buffer.cpp',
+  include_directories : include_directories, 
+  dependencies : [fmt_dep, ut_dep],
+  install: false)
+test('qa_buffer', buffer_test_exe)
+
+ut_test_names = [
+    'qa_dynamic_port',
+]
+deps = [fmt_dep, ut_dep, reflcpp_dep]
+cpp_args = '-fsanitize=address'
```

Comment:
Can you just use meson's builtin support for `-Db_sanitize=address` here?

You don't *have* to apply it to the entire project, by the way. Applying it to a single target as `override_options: ['b_sanitize=address']` works.

Reply:
Thanks for the feedback @eli-schwartz !